tutorials/clustering.md

```diff
@@ -0,0 +1,15 @@
+# Scaling out across multiple nodes
+
+>:WARNING: Running TimescaleDB in a multi-node setup is currently in PRIVATE BETA.
+This method of deployment is not meant for production use.
+
+Now that you have installed TimescaleDB and know how to create hypertables,
+we are going to go ahead and create a new distributed hypertable.
+
+First, you need to create a distribute database that contains your distributed
+hypertable.
+
+1. TimescaleDB installed on at least 2 nodes. In this tutorial, we will
```

Comment:
confusing language about needed 2 nodes but spinning up on 3?  I think there is somethign about wanting to divide data across 2 data nodes, so spinning up three nodes (2 for data, one to serve as access node?)

Reply:
This is out of scope here - also, the PR has been updated and doesn't look like this anymore.